docs/actions/bazarliste.yaml

```
@@ -225,6 +225,16 @@ actions:
         showif: facettes
         showExceptFor:
           - bazargogocarto
+      resetfiltersbutton:
+        label: _t(AB_bazar_commons2_resetfiltersbutton_label)
+        advanced: true
+        type: checkbox
+        default: 0
+        checkedvalue: 0
```

```suggestion
        checkedvalue: 1
```
J'ai une question : qu'est-ce qui t'a fat changé la valeur par défaut de 1 à 0 ?

---

Oui c'était suite à une série de test, merci pour ta vigilance. J'avais simplement pas fait gaffe que j'avais remis en inversé puisque le filtre s'active et se désactive bien quand même (il a parfois des comportements bizarre cet action builder). Mais logiquement c'est bien dans le sens que c'était.

Tu pourras remarquer que j'ai amendé du coup mon dernier commit pour enlever cette modification.